File: problems/2032-largest-odd-number-in-string/largest-odd-number-in-string.js
/**
 * @param {string} num
 * @return {string}
 */
const largestOddNumber = (num) => {
    for (let i = num.length - 1; i >= 0; i--) {
        if (num[i] % 2 !== 0) {
            return [...num].splice(0, i + 1).join("");
        }
    }
    return "";
};

File: problems/2032-largest-odd-number-in-string/largest-odd-number-in-string.py
class Solution:
    def largestOddNumber(self, num: str) -> str:
        for i in range(len(num) - 1, -1, -1):
            if int(num[i]) % 2 != 0:
                tmp = num[0 : i + 1]    
                return "".join(tmp)
            
        return ""